bin/fasta-to-phylip.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+
+import sys
+import tempfile
+
+from dark.fasta import FastaReads
+from dark.fasta_ss import SSFastaReads
+from dark.fastq import FastqReads
+
+
+if __name__ == '__main__':
+    import argparse
+
+    parser = argparse.ArgumentParser(
+        description=('Given FASTA on stdin write the sequence ids and lengths '
+                     'to stdout.'))
+
+    parser.add_argument(
+        '--readClass', default='fasta', choices=('fasta', 'fastq', 'fasta-ss'),
```

Comment:
OK, I opened a ticket. https://github.com/acorg/dark-matter/issues/491 I'd do it now on these scripts but then we'd have 2 different ways of naming the same thing and someone would have to go fix the other 4 instances and find all calls, etc. That's the kind of thing that sometimes never happens... I'd rather have a ticket to fix them all.